test/Spec/BotPlutusInterface/Contract.hs

```diff
@@ -823,19 +820,20 @@ useWriter = do
   let txOutRef = TxOutRef "e406b0cf676fc2b1a9edb0617f259ad025c20ea6f0333820aa7cef1bfe7302e5" 0
       txOut = TxOut pkhAddr1 (Ada.lovelaceValueOf 1200) Nothing
       initState = def & utxos .~ [(txOutRef, txOut)]
+      test = "test"
 
       contract :: Contract (Last Text) (Endpoint "SendAda" ()) Text CardanoTx
       contract = do
         tell $ Last $ Just "Init contract"
         let constraints =
               Constraints.mustPayToPubKey paymentPkh2 (Ada.lovelaceValueOf 1000)
         txId <- submitTx constraints
-        tell $ Last $ Just $ Text.pack $ show $ Tx.txId <$> txId
+        tell $ Last $ Just test
```

Comment:
[nits] The purpose of returning the txId was to prove that we can return values from the contract, but I think it is not really important to include in this test case.
We could just assert that the value in the observable state is "Init contract".